### DISCUSSION TITLE
Tailwind Table Row that spans multiple columns

### DISCUSSION THREAD

**Discussion by myhendry** [tailwindlabs/tailwindcss#7846] [Help]
Hi. How do we design a tailwind table with a specific row that spans x number of columns without affecting the other rows? Thanks

```

  <table className="table-fixed border w-full">
          <tbody>

            <tr>
              <td className="p-1">Name: {data.name}</td>
            </tr>

            <tr>
              <td className="border p-1">Age: {data.age}</td>
            </tr>

            <tr className="border font-semibold p-2 text-justify">
              <td className="p-2">
                  {data.address}
              </td>
            </tr>

          </tbody>
        </table>
```

**Comment by CharlesKasasira** [comment]
Using the HTML attribute `colSpan` helped me with this.

```
<td colSpan={6} className="w-full left-0 bg-gray-100 p-2 -bottom-10"></td>
```

**Reply by zeekrey** [reply]
I'm on the same track, but using colSpan makes it challenging to add media queries, or am I wrong? I've built a large, complex table layout based on numerous col/row spans, and now I'm considering whether a grid might have been a better option.
